.github/workflows/release-tag.yml

```diff
@@ -108,8 +117,6 @@ jobs:
           path: |-
             ghostty-${{ env.GHOSTTY_VERSION }}.tar.gz
             ghostty-${{ env.GHOSTTY_VERSION }}.tar.gz.minisig
-            ghostty-source.tar.gz
-            ghostty-source.tar.gz.minisig
```

Comment:
These were from the 1.0 release and were generally a mistake to make, the `ghostty-VERSION` builds are the recommended and only documented ones at this point. I kept the old ones around for a time for build tooling but everything should be updated by now.